content/coronavirus_landing_page.yml

```diff
@@ -2,17 +2,8 @@ content:
   title: "Coronavirus (COVID-19): guidance and support"
   meta_description: "Find information on coronavirus, including guidance, support, announcements and statistics."
   page_header: "Coronavirus (COVID&#8209;19)"
+  # The header section is edited in https://collections-publisher.publishing.service.gov.uk/coronavirus/landing
   header_section:
```

Comment:
I think this line can be removed too.

Reply:
@leenagupte I kept it in because `announcements:` and `sections:` still have the key underneath the comment.

Reply:
Which is also odd. But I think `announcements:` and `sections:` used to contain lists so were a bit different.

Reply:
thanks, have removed it. Will make a Trello card to clean up unused keys.